README.md

```diff
@@ -138,7 +138,7 @@ Refer to [the documentation](DOCUMENTATION.md) to learn more about supported con
 * Cluster-wide unique ID generator, called **FlakeIdGenerator**
 * Distributed, CRDT based counter, called **PNCounter**
 * Distributed concurrency primitives from CP Subsystem such as **FencedLock**, **Semaphore**, **AtomicLong**
-* Integration with [Hazelcast Cloud](https://cloud.hazelcast.com/)
+* Integration with [Hazelcast Cloud](https://viridian.hazelcast.com/)
```

Comment:
```suggestion
* Integration with [Hazelcast Viridian](https://viridian.hazelcast.com/)
```

Reply:
This file hasn't changed, it was a Github glitch it was showing. I'm going to leave this as out of scope.